### PR TITLE
Add support for AWS Athena

### LINKED ISSUE #6: Amazon Athena integration

### PR DESCRIPTION
**Pull request by aleDsz**
https://user-images.githubusercontent.com/6402997/173854556-134e1e5e-c675-4437-8d92-d93ed837d442.mp4

- Closes #6

### CODE REVIEW

**Comment by josevalim** [comment]
Do we need to specify the S3 location? Or does it have a reasonable default?

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+elixir 1.13.4
```
Comment (jonatanklosko) [inline .tool-versions:1]: Unintended I believe :cat2:

**Comment by jonatanklosko** [comment]
The code looks good to me! :)

File: mix.exs [hunk 2]
```diff
@@ -33,6 +33,7 @@ defmodule KinoDB.MixProject do
       {:myxql, "~> 0.6.2 or ~> 0.7", optional: true},
       {:db_connection, "~> 2.4.2", optional: true},
       {:req_bigquery, github: "livebook-dev/req_bigquery", optional: true},
+      {:req_athena, path: "~/github/livebook-dev/req_athena", optional: true},
```
Comment (jonatanklosko) [inline mix.exs:36]: Please make sure to point to github before merging :)

File: .tool-versions (hunk 1 above)
Comment (aleDsz) [reply .tool-versions:1]: yeah, it was 😬

**Comment by aleDsz** [comment]
> Do we need to specify the S3 location? Or does it have a reasonable default?

Sadly yes, even the API doesn't mention that's required, if we try to not send they return a status 400 with this error: `No output location provided. An output location is required either through the Workgroup result configuration setting or as an API input.`

**Pull request merged**